### DISCUSSION TITLE
Strix 15 G533 (2022) Missing M5 Keyboard

### DISCUSSION THREAD

**Discussion by RaZer0k** [seerge/g-helper#945] [General]
Asus Strix 15 G533 (2022) got five custom keys 'M', but G-Helper only can handler four...

https://rog.asus.com/laptops/rog-strix/rog-strix-scar-15-2022-series/

Can you add the missing one?

**Comment by seerge** [comment]
It already handles all of them, M4 on Strix is Fn+F5 for others

**Reply by Keethos** [reply]
![SCAR M4 Photo](https://github.com/seerge/g-helper/assets/166189975/590a4ea7-e95c-49fd-ae5a-6c895289e2b1)
![SCAR M4](https://github.com/seerge/g-helper/assets/166189975/dd3ac77f-9640-4477-a0da-0d65bf72d960)
Hi,

Sorry to dig up this old thread but it's the closes thread to the issue I'm having with my recently purchased STRIX SCAR 16.  Hopefully you can see from the attached images that G-Helper can see the M1-M3 and M5 buttons but does not see M4 for some reason, which is annoyingly the one button I wish to remap since FN+F5 does exactly the same thing (and not really used yet).  You mentioned that M4 on Strix is FN+F4 but that's not the case with my laptop, this is the Toggle Aura option, changing this does not effect the M4 button at all.

Is there some way for me to enable the remapping of the M4 button?  Am I missing something since no one else has mentioned this?

Thanks in advanced.

**Reply by seerge** [reply]
@Keethos hello, I was answering this numerous of times.

On Strix models M4 and FN+F5 produce exactly same code.  So it's impossible to separate them in bindings.

And this is reserved by the app for switching profiles forwards and backwards. Therefore there is no reason to rebind it as you can't separate it from FN+F5

**Reply by Keethos** [reply]
Ah ok, shame, but thanks for the swift response.
